tests/test_app.py

```diff
@@ -236,128 +228,78 @@ def test_new_stack(monkeypatch, app, mock_senza, oauth_requests):
+                                         '2b', '1.0', ['abc', 'def'], False,
                                          {'LizzyTargetTraffic': 100,
                                           'LizzyKeepStacks': 0})
     assert request.status_code == 201
     assert request.headers['X-Lizzy-Version'] == CURRENT_VERSION
-    stack_version = FakeStack.last_save['stack_version']
-    assert FakeStack.last_save['application_version'] is None
-    assert FakeStack.last_save['image_version'] == '1.0'
-    assert FakeStack.last_save['keep_stacks'] == 0
-    assert FakeStack.last_save['parameters'] == ['abc', 'def']
-    assert FakeStack.last_save['stack_id'] == 'abc-' + stack_version
-    assert FakeStack.last_save['stack_name'] == 'abc'
-    assert FakeStack.last_save['status'] == 'CF:CREATE_IN_PROGRESS'
-    assert FakeStack.last_save['traffic'] == 100
 
     data = {'keep_stacks': 0,
             'new_traffic': 100,
             'image_version': '1.0',
+            'stack_version': '42',
             'senza_yaml': 'SenzaInfo:\n  StackName: abc',
             'parameters': ['abc', 'def'],
             'application_version': '42'}
     mock_senza.reset_mock()
     mock_senza.render_definition.return_value = GOOD_CF_DEFINITION
-    mock_kio.versions_create.return_value = True
-    request = app.post('/api/stacks', headers=GOOD_HEADERS, data=json.dumps(data))  # type: flask.Response
+    request = app.post('/api/stacks', headers=GOOD_HEADERS,
+                       data=json.dumps(data))  # type: flask.Response
     mock_senza.assert_called_with('eu-west-1')
     mock_senza.create.assert_called_with('SenzaInfo:\n  StackName: abc',
-                                         ANY, '1.0', ['abc', 'def'], False,
+                                         '42', '1.0', ['abc', 'def'], False,
                                          {'LizzyKeepStacks': 0,
                                           'LizzyTargetTraffic': 100})
-    mock_kio.assert_called_with()
-    mock_kio.versions_create.assert_called_once_with(application_id='abc',
-                                                     artifact='pierone.example.com/lizzy/lizzy:12',
-                                                     version='42')
     assert request.status_code == 201
     assert request.headers['X-Lizzy-Version'] == CURRENT_VERSION
-    assert FakeStack.last_save['application_version'] == '42'
-    assert FakeStack.last_save['image_version'] == '1.0'
-    assert FakeStack.last_save['keep_stacks'] == 0
-    assert FakeStack.last_save['parameters'] == ['abc', 'def']
-    assert FakeStack.last_save['stack_id'] == 'abc-42'
-    assert FakeStack.last_save['stack_name'] == 'abc'
-    assert FakeStack.last_save['status'] == 'CF:CREATE_IN_PROGRESS'
-    assert FakeStack.last_save['traffic'] == 100
-
-    mock_kio.reset_mock()
+    request_data = json.loads(request.data.decode())
+    assert request_data == {'creation_time': '2016-04-14T11:59:27+00:00',
```

Comment:
I think it does not matter much, but `creation_time` field is a timestamp/integer now.